R/generate_subsample.R

```diff
@@ -37,18 +37,36 @@
 #' * **subsamples**
 #' @export
 #' @md
-generate_subsample <- function(con, season) {
+generate_subsample <- function(con, sampling_event, season) {
```

Comment:
are these updates meant to be part of this PR?

Reply:
No - I must have checked out this branch from the subsample branch. that's my fault. It might be easier if we review/merge that branch first and then merge this one in?